includes/class-gam.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Newspack Ads GAM Integration.
+ *
```

Comment:
Would you add a more descriptive description of the purpose of this integration please?

Reply:
Added in 23e9e3674d5988920cd063b6acfe3e0d61fff76c